src/roswire/ros1/launch/reader.py

```diff
@@ -372,7 +372,11 @@ def _load_node_tag(
 
         # locate node executable and determine the type
         executable_path = self.locate_node_binary(package, node_type)
-        executable_type = self._get_executable_type(executable_path)
+        if executable_path != "\\unknown":
```

Comment:
this seems unrelated?

Reply:
yeah, you're right.

Reply:
Changed comment to reflect this additional change.